crates/vision/src/image_segmenter.rs

```diff
@@ -165,30 +165,9 @@ impl ScanLineState {
     }
 }
 
-fn median_of_three(values: [u8; 3]) -> u8 {
-    let [first, second, third] = values;
-    // TODO: replace with same approach as median_of_five()
-    if first <= second {
-        if second <= third {
-            // first <= second <= third
-            second
-        } else if first <= third {
-            // first <= third < second
-            third
-        } else {
-            // third < first <= second
-            first
-        }
-    } else if first <= third {
-        // second < first <= third
-        first
-    } else if second <= third {
-        // second <= third < first
-        third
-    } else {
-        // third < second <= first
-        second
-    }
+fn median_of_three(mut values: [u8; 3]) -> u8 {
+    let (_, median, _) = values.select_nth_unstable(1);
+    *median
 }
```

Comment:
Should this maybe be separate? This doesn't seem like just parameters to me

Reply:
Yes, but it's also related to clean up and I think it doesn't deserve it's own pr...

Reply:
Removed